### PR TITLE
Enable Source Link

### LINKED ISSUE #18: Please include sources in Nuget package
Hi Roman, thank you for the great library.

On my machine, using Rider and Nuget vir Paket, I can't see the F# source code (I get decompiled C#). With some other packages, like FsToolkit.Errorhandling, I can see the F# source. 

Apparently this is something to do with the way it was packaged - I found this page describing source link for Nuget packages: https://devblogs.microsoft.com/nuget/introducing-source-code-link-for-nuget-packages/

It would be great if you could include a link to the source when you package up the next.

Thanks again for the great library.

Roland

### PR DESCRIPTION
**Pull request by randrag**
Hi Roman, 

I've made the changes, built it on my local machine, and tested that the sources are available to a project referencing the newer version.

It took me a while to figure out that I had to commit changes and push to the remote repo on Github before building and staging the nuget file, in order for Source Link to pick up the source code, since it fetches the source from the specific commit in the repo.

Hopefully this is satisfactory and closes #18.

Thanks again for a great library,

Roland

### CODE REVIEW

File: src/Dapper.FSharp/Dapper.FSharp.fsproj [hunk 1]
```diff
@@ -25,5 +30,10 @@
     <Compile Include="Builders.fs" />
     <Compile Include="OptionTypes.fs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
```
Comment (Dzoukr) [inline src/Dapper.FSharp/Dapper.FSharp.fsproj:35]: Can we use normal paket.dependencies + paket.references files for this or is it necessary to have it in fsproj? 🤔

**Comment by randrag** [comment]
I will test and revert.

**Comment by randrag** [comment]
Hi Roman, it looks like it is sufficient to just add the nuget reference to the build section of the paket.dependencies file.

One side note is that I don't know where the `PrivateAssets="All"` goes in this scenario. In any case, the link seems to work fine from my machine.

**Comment by Dzoukr** [comment]
No problem Roland, I'll merge and add it. I found forki's comment about PrivateAssets here: https://github.com/fsprojects/Paket/issues/2951#issuecomment-354417950

**Comment by Dzoukr** [comment]
Thanks again, Roland. Will release soon!

**Comment by Dzoukr** [comment]
Aaaaaaaand released! In your Nuget feed soon.

**Comment by randrag** [comment]
Thank you, just ran paket update in our production project, and it works!

**Pull request merged**